bin/compile

```diff
@@ -143,7 +144,31 @@ export CPPPATH="$INCLUDE_PATH"
 export PKG_CONFIG_PATH="$BUILD_DIR/.apt/usr/lib/x86_64-linux-gnu/pkgconfig:$BUILD_DIR/.apt/usr/lib/i386-linux-gnu/pkgconfig:$BUILD_DIR/.apt/usr/lib/pkgconfig:$PKG_CONFIG_PATH"
 
 #give environment to later buildpacks
-export | grep -E -e ' (PATH|LD_LIBRARY_PATH|LIBRARY_PATH|INCLUDE_PATH|CPATH|CPPPATH|PKG_CONFIG_PATH)='  > "$LP_DIR/export"
+topic "Exporting environment"
+BUILDPACK_EXPORT_FILE="$BP_DIR/export"
+export | grep -E -e ' (PATH|LD_LIBRARY_PATH|LIBRARY_PATH|INCLUDE_PATH|CPATH|CPPPATH|PKG_CONFIG_PATH)=' > "$BUILDPACK_EXPORT_FILE"
 
 topic "Rewrite package-config files"
 find "$BUILD_DIR/.apt" -type f -ipath '*/pkgconfig/*.pc' -print0 | xargs -0 --no-run-if-empty -n 1 sed -i -e 's!^prefix=\(.*\)$!prefix='"$BUILD_DIR"'/.apt\1!g'
+
+# post-installation logic for git
+# check is git is in the Aptfile and set GIT_EXEC_PATH
+if grep -qw "git" "${BUILD_DIR}/Aptfile"; then
+  if command -v git >/dev/null 2>&1; then
```

Comment:
This seems like the grep might be at risk of false positives?

I wonder if a safer way to check if the apt buildpack installed git would be to remove both the grep and this current check, and instead replace it with a check of the path returned by `command -v git` to see if it matches the APT buildpack's install path? Or even simply a `[[ -f "${BUILD_DIR}/.apt/usr/bin/<whatever>" ]]` etc?

Reply:
Fixed in https://github.com/heroku/heroku-buildpack-apt/pull/143/commits/e6f7e4f89f4e847b0ac2d8440ed69716d484b96e